File: Talleres-2B/Taller-1/TallerLab2B1S/src/tallerlab2b1s/NumDentroVector.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package tallerlab2b1s;
import java.util.Scanner;
/**
 *
 * @author Santiago Espinoza
 */
public class NumDentroVector {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        
        //Declaración e inicialización de las Variables 
        int i, n, num, cont;
        i = 0;  n= 0; num = 0; cont = 0;
        
        //Declaración e Inicialización
        System.out.println("\t ----CONTAR Y BUSCAR UN NÚMERO DENTRO DE UN VECTOR----\n");
        System.out.println("¿Cuantos números van a ser ingresados?");
        n = leer.nextInt();
        
        //Declaración del vector
        int [] A = new int [n];
        
        //Ingreso de los elementos del vector
	for (i=0; i<n; i++){
            System.out.println("Ingrese el elemento A[" +i +"]: ");
            A[i]= leer.nextInt();
        }
        
        //Ingreso del número
        System.out.println("Ingrese el número a verificar");
        num = leer.nextInt();
        
        //Proceso 
        for (i=0; i<n; i++){
            if (num == A[i]){
                cont = cont + 1;
                System.out.println("El número " +num +" se encuentra en la posición: A[" +i +"] \t");
            }
        }
        System.out.println("\nEl número " +num +" se repetio " +cont +" veces"); 
    }
}

File: Talleres-2B/Taller-1/NumDentroVector.py
print("\t---CONTAR Y BUSCAR UN NÚMERO DENTRO DE UN VECTOR---")

n = int(input("¿Cuantos elementos va a tener su vector?: "))

#Declaración del vector
A = []

#Ingreso de los elementos del vector
for i in range(n):
    x = int(input("Ingrese el elemento: "))
    A.append(x)

#Ingreso del número
num = int(input("Ingrese el número a verificar: "))

#proceso
cont = 0
for i in range(n):
    if num == A[i]:
        cont = cont + 1
        print("El número ",num," se encuentra en la posición: A[",i,"]")
print("\nEl número ",num," se repetio ",cont," veces")